src/components/PostCard.tsx

```diff
@@ -91,26 +106,12 @@ const PostCard: React.FC<PostCardProps> = ({
           {post.content}
         </p>
       </div>
-      {isHosted && (
-        <div
-          className={`absolute bottom-0 right-0 px-4 py-6 w-full flex justify-between gap-4`}
-        >
-          <button
-            type="button"
-            onClick={e => handleAdminBtnClick(e)}
-            className="btn btn-second btn-sm font-bold z-100 flex-1"
-          >
-            모집글 보기
-          </button>
-          <button
-            type="button"
-            onClick={e => handleAdminBtnClick(e)}
-            className="btn btn-second btn-sm font-bold z-100 flex-1"
-          >
-            신청자 보기
-          </button>
-        </div>
-      )}
+      <PostCardBtn
+        post={post}
+        isHosted={isHosted}
+        isParticipated={isParticipated}
+        status={status}
+      />
```

Comment:
PostCardBtn은 마이페이지에서만 필요한데 다른곳(메인페이지)에서도 렌더링하게 되어있습니다. 지금 isHosted를 props로 넘겨서 jsx렌더링 여부를 결정하는데에만 사용하고 있는데
그냥 여기에서 렌더링여부를 결정하는게 더 맞는 것 같습니다
```suggestion
      {isHosted && <PostCardBtn
        post={post}
        isParticipated={isParticipated}
        status={status}
      />}
```